src/main/java/com/myFirstProject/myFirstProject/service/BasketServiceImpl.java

```diff
@@ -39,18 +39,28 @@
     public Long saveBasket(BasketReq basketReq) {
         logger.info("Get order to update {}", basketReq);
         sumValidation(basketReq);
+        promoCodeValidation(basketReq);
```

Comment:
wrong name
promoCodeValidation - it is not method name style. it should be like a verb - validatePromoCode